maven/tests/all_tests.bzl

```diff
@@ -1,14 +1,33 @@
-load(":dicts_test.bzl", dicts = "dicts_test_suite")
-load(":paths_test.bzl", paths = "paths_test_suite")
-load(":poms_test.bzl", poms = "poms_test_suite")
-load(":sets_test.bzl", sets = "sets_test_suite")
-load(":strings_test.bzl", strings = "strings_test_suite")
+load(":dicts_test.bzl", dicts = "suite")
+load(":paths_test.bzl", paths = "suite")
+load(":maven_test.bzl", maven = "suite")
+load(":poms_test.bzl", poms = "suite")
+load(":sets_test.bzl", set_tests = "suite")
+load(":strings_test.bzl", strings = "suite")
 load(":xml_test.bzl", xml = "suite")
 
-def _all_tests_rule_impl(ignore):
-    for suite in [dicts, paths, poms, sets, strings, xml]:
+load("//maven:sets.bzl", "sets")
+
+
+SUITES = [dicts, paths, maven, poms, set_tests, strings, xml]
+
+def _validate(ctx, suites):
+    # Function objects don't have good properties, so we hack it from the string representation.
```

Comment:
Seems like I'm seeing overlap btw this PR and #28. Happy merging.

Reply:
Yeah.  I have a merge commit I was using, so I'll just fold it in when I rebase.  I factored the test PR out of this one.